lib/src/over_react_test/react_util.dart

```diff
@@ -446,8 +446,8 @@ Element getComponentRootDomByTestId(dynamic root, String value, {String key = de
 ///     queryByTestId(renderedInstance, 'value'); // returns the `inner` `<div>`
 ///
 /// Related: [queryAllByTestId], [getComponentRootDomByTestId].
-Element queryByTestId(dynamic root, String value, {String key = defaultTestIdKey}) {
-  return findDomNode(root).querySelector('[$key~="$value"]');
+Element queryByTestId(dynamic root, String value, {String key = defaultTestIdKey, bool searchInShadowDom = false}) {
```

Comment:
Why is there no `shadowDepth` argument here?

Reply:
lol i legit had this fixed and apparently never commited it.